client/packages/common/src/hooks/useNativeClient/useNativeClient.ts

```diff
@@ -22,7 +23,7 @@ export type FrontEndHost = {
 
 export interface NativeAPI {
   // Method used in polling for found servers
-  discoveredServers: () => Promise<FrontEndHost[]>;
+  discoveredServers: () => Promise<{ servers: FrontEndHost[] }>;
```

Comment:
looked like capacitor deals with object, couldn't find a way to send array as per before

Reply:
ah, right. I see - I was reading from bottom up, and this explains something I was wondering about.